Src/Support/Google.Apis.Auth/GoogleJsonWebSignature.cs

```diff
@@ -230,12 +230,6 @@ public class Header : JsonWebSignature.Header
         /// </summary>
         public class Payload : JsonWebSignature.Payload
         {
-            /// <summary>
```

Comment:
We can't remove this property - it's a breaking change in a public class.

If it shouldn't be here, mark it as obsolete instead.

Reply:
This was a mistake, I have almost defeated the purpose of this class :) reverting